package.json

```diff
@@ -24,6 +24,7 @@
   "devDependencies": {
     "@babel/preset-env": "^7.20.2",
     "@parcel/transformer-sass": "^2.8.1",
+    "buffer": "^5.5.0",
```

Comment:
Did this creep in from another branch, or something? I'm not sure what you'd use it for in this set of changes.